libcore/Directory.vala

```diff
@@ -860,7 +860,16 @@ public class Files.Directory : Object {
         file_hash.insert (gof.location, gof);
     }
 
-    private Files.File file_cache_find_or_insert (GLib.File file, out bool was_in_hash, bool update_hash = false) {
+    private Files.File? file_cache_find_or_insert (
+        GLib.File file,
+        out bool was_in_hash,
+        bool update_hash
```

Comment:
How come you want to remove the default false value here?

Reply:
Not sure now, but I am not in general a fan of default parameters. I probably wanted to ensure a conscious decision from the dev in this case.